handler.go

```diff
@@ -431,28 +434,37 @@ func (fm *Frontman) processInputContinuous(inputFilePath string, local bool, int
 	lastFetch := time.Unix(0, 0)
 	interval := secToDuration(fm.Config.Sleep)
 
+	waitWg := sync.WaitGroup{}
+
+	logrus.Infof("STARTING processInputContinuous")
+
 	for {
+		select {
+		case <-interrupt:
```

Comment:
instead of interrupt you could use <-context.Done() as it's a recommended go-way to cancel running subprocesses and requests

Reply:
This is just recycled code from the refactor.

Reply:
sure